ethereumj-core/src/main/java/org/ethereum/solidity/compiler/Solc.java

```diff
@@ -25,6 +27,22 @@ private Solc() {
     }
 
     private void init() throws IOException {
+        SystemProperties defaultProps = SystemProperties.getDefault();
+        if (defaultProps != null && defaultProps.customSolcPath() != null) {
+            solc = new File(defaultProps.customSolcPath());
+            solc.setExecutable(true);
```

Comment:
Do you think this call is necessary. Shouldn't it generate exception normally? (I mean solc normally should have root modification privilege)